skymp5-server/cpp/papyrus_vm_lib/ActivePexInstance.cpp

```diff
@@ -710,7 +711,7 @@ uint8_t ActivePexInstance::GetTypeByName(std::string typeRef)
     return VarValue::kType_ObjectArray;
   }
   if (typeRef == "none") {
-    // assert(false);
+   // throw std::runtime_error("Error");
```

Comment:
I think that at this point it's better to just remove all such commented code altogether. `blame` shows that it's almost as old as this repo is, and well.. even if they were originally meant not to forget something, everybody has forgotten that anyway